llvm/lib/MC/MCTargetOptionsCommandFlags.cpp

```diff
@@ -140,6 +141,10 @@ llvm::mc::RegisterMCTargetOptionsFlags::RegisterMCTargetOptionsFlags() {
       cl::init(true));
   MCBINDOPT(X86RelaxRelocations);
 
+  static cl::opt<bool> X86Sse2Avx(
+      "x86-sse2avx", cl::desc("Convert SSE Instructions to AVX Instructions"));
```

Comment:
Use the same help info as driver and GNU assembler.

Specify that the assembler should encode SSE instructions with VEX prefix

Reply:
Done